memory-db/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 
 [dependencies]
 heapsize = { version = "0.4", optional = true }
-parity-util-mem = { version = "0.6", default-features = false }
+parity-util-mem = { version = "0.7.0", default-features = false }
 hash-db = { path = "../hash-db", default-features = false, version = "0.15.2"}
 hashbrown = { version = "0.6.3", default-features = false, features = [ "ahash" ] }
```

Comment:
In #94 an upgrade to hashbrown 0.8 was needed, it could be the same here.

Reply:
Hmm, interesting. This works fine in openethereum without any other upgrades.

Reply:
Yes hashbrown is mainly use for no_std use of memorydb, so it should not change for eth, not sure it will break on substrate either (only if util-mem with no_std is use which I doubt), but still is the util-mem current version.
I also see that the feature "hashbrown" for was enable on the pr, I do not entirely remember why, I will try to use master on substrate to remember.

Reply:
Does not looks needed in substrate either (but kvdb-web need an update though, I created https://github.com/paritytech/parity-common/pull/404)

Reply:
So I can resolve this then? :)

Reply:
Yes, upgrading hashbrown could be done later.